### PR TITLE
Cleanup unused/outdated dependencies & Fix .env loading issue

### PR DESCRIPTION
**Pull request by ngmachado**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,28 +20,26 @@
...
     "ethereumjs-wallet": "^1.0.1",
     "express": "^4.17.1",
-    "pm2": "^5.1.0",
-    "sequelize": "^6.6.2",
-    "sqlite3": "^5.0.2",
-    "web3": "1.5.2",
-    "winston": "^3.3.3",
-    "winston-array-transport": "^1.1.4"
+    "sequelize": "^6.6.5",
+    "sqlite3": "^4.2.0",
```
Comment (d10r) [inline package.json:31]: why was sqlite3 downgraded?

**Comment by ngmachado** [comment]
The downgrade was made by using `npm audit --fix`

```
Depends on vulnerable versions of tar
  node_modules/node-gyp
    sqlite3  >=5.0.0
    Depends on vulnerable versions of node-gyp
    node_modules/sqlite3
```

File: src/app.js [hunk 2]
```diff
@@ -122,7 +123,19 @@ class App {
...
-            this.logger.debug(JSON.stringify(this.config.getConfigurationInfo()));
+            const userConfig = this.config.getConfigurationInfo();
+            this.logger.debug(JSON.stringify(userConfig));
+            //check important change of configurations
+            const res = await this.db.queries.getConfiguration();
+            if(res !== null) {
+                const dbuserConfig = JSON.parse(res.config)
+                if(dbuserConfig.LISTEN_MODE !== userConfig.LISTEN_MODE && userConfig.LISTEN_MODE == 1) {
+                    this._needResync = true;
+                    this.logger.error(`ATTENTION: LISTEN_MODE changed from the last boot, please resync the database`);
```
Comment (d10r) [inline src/app.js:134]: This is imo the wrong check.  
What we actually want to check is if the set of tokens to monitor has changed. This may or may not be caused by LISTEN_MODE having changed.   
I also suggest to rename LISTEN_MODE to something like ONLY_LISTED_TOKENS (inverted logic - so it would become false by default).

File: test/integration/gas.integration.test.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ const bootNode = async (delayParam = 0) => {
         epoch_block: 0,
         DB: "TestDatabase.sqlite",
         protocol_release_version: "test",
-        tx_timeout: 300000,
+        tx_timeout: 20,
```
Comment (d10r) [inline test/integration/gas.integration.test.js:25]: why was this changed only in this test and not in the others too?
